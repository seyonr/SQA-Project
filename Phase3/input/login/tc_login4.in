login 
standard
12345
aaJimmy