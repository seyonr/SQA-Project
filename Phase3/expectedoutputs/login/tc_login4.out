Welcome to the banking system!
Do you want to login or exit : 
Enter login type (admin or standard): 
Enter account number: 
Enter account name: 
Invalid account name, session terminated!
